spec/features/post_a_message_to_Chitter_spec.rb

```diff
@@ -0,0 +1,9 @@
+feature 'Post a message to Chitter' do
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)